addons/mass_mailing/static/src/models/trace.js

```diff
@@ -0,0 +1,123 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core'
+import { attr } from '@mail/model/model_field'
+
+registerModel({
+    name: 'MailingTrace',
+    identifyingFields: ['id'],
```

Comment:
note: after rebase master syntax has changed, now just have `identifying: true` on the field directly

Reply:
Noted, will apply all these in next push, thanks for the review :)